test/kt/CMakeLists.txt

```diff
@@ -122,3 +123,50 @@ if (ONEDPL_TEST_ENABLE_KT_ESIMD)
     # Pin some cases to track them, e.g. because they fail
     _generate_esimd_sort_test("256" "32" "double" "" 1000) # segfault
 endif()
+
+function (_generate_igpu_scan_test _data_per_work_item _work_group_size _type)
+
+    if ((NOT TARGET build-igpu-tests) AND (NOT TARGET run-igpu-tests))
+        add_custom_target(build-igpu-tests COMMENT "Build all scan kernel template tests")
+        add_custom_target(run-igpu-tests
+            COMMAND "${CMAKE_CTEST_COMMAND}" -R "^run-igpu-tests$" --output-on-failure --no-label-summary
+            DEPENDS build-igpu-tests
```

Comment:
I changed it to `build-scan-kt-tests` since a target called `build-gpu-tests` sounds ambiguous to me (i.e., it sounds like it should encompass more than just the scan kernel template tests, which is not the intent).